mobile/src/main/res/xml/preferences.xml

```diff
@@ -58,6 +58,38 @@
             android:summary="@string/settings_openhab_sslhost_summary"
             android:title="@string/settings_openhab_sslhost" />
     </PreferenceCategory>
+    <PreferenceCategory android:title="Presence">
+        <CheckBoxPreference
+            android:defaultValue="false"
+            android:key="default_openhab_presence_enable"
+            android:summary="@string/presence_pref_enable_summary"
+            android:title="@string/presence_pref_enable_title" />
+        <EditTextPreference
+            android:defaultValue="isPresent"
+            android:dependency="default_openhab_presence_enable"
+            android:inputType="textNoSuggestions"
+            android:key="default_openhab_presence_item"
+            android:summary=""
+            android:title="@string/presence_pref_item_title" />
+        <EditTextPreference
+            android:dependency="default_openhab_presence_enable"
+            android:inputType="numberDecimal"
+            android:key="default_openhab_presence_lat"
+            android:summary=""
+            android:title="@string/presence_pref_lat_title" />
+        <EditTextPreference
+            android:dependency="default_openhab_presence_enable"
+            android:inputType="numberDecimal"
+            android:key="default_openhab_presence_lng"
+            android:summary=""
```

Comment:
Can you display the selected value as summary?

Reply:
That's a good idea, done!